src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+import core.basesyntax.converter.ConverterCSVToObject;
+import core.basesyntax.converter.ConverterObjectToCSV;
+import core.basesyntax.service.FruitOperationGenerator;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException, CsvRequiredFieldEmptyException, CsvDataTypeMismatchException {
+        String inputFileName = "src/main/resources/fruitShop.csv";
+        ConverterCSVToObject converterCSVToObject = new ConverterCSVToObject();
```

Comment:
move methods from ConverterCSVToObject and ConverterObjectToCSV to common class FileUtils (in package 'util') and make them static.

Reply:
done